python/requirements.txt

```diff
@@ -1,11 +1,8 @@
-black==23.3.0
-flake8==6.0.0
```

Comment:
Why are we removing black and flake8?  Are these managed by tox in venvs?

Reply:
yeh, specifically the `format` and `lint` venvs configured in the `tox.ini` file. I can add them back to requirements but they're only needed for development and not to build a distribution